lib/templates/bootstrap/datepicker.android.js

```diff
@@ -1,6 +1,6 @@
-var React = require("react")
-var { View, Text, TouchableNativeFeedback } = require("react-native")
-var { DateTimePickerAndroid } = require("@react-native-community/datetimepicker")
+var React = require('react')
```

Comment:
why single quotes all of a sudden?

Reply:
This is a forked version of `tcomb-form-native` that we made tweaks a long time ago to keep running. This round is another bump to keep things rolling. This library has an eslint rule for single quotes, but eslint wasn't actually running so it wasn't being caught. I changed the few instances of double quotes to single quotes to limit the breadth of the changes.